OpenRA.Mods.Common/Widgets/Logic/Ingame/ObserverStatsLogic.cs

```diff
@@ -325,5 +357,44 @@ class StatsDropDownOption
 			public Func<bool> IsSelected;
 			public Action OnClick;
 		}
+
+		public bool HandleKeyPress(KeyInput e)
+		{
+			if (e.Event == KeyInputEvent.Down && !e.IsRepeat)
+			{
+				var h = Hotkey.FromKeyInput(e);
+				if (h == statisticsBasicKey.GetValue())
```

Comment:
If you put these keys in an array then you could reduce duplication by looping over them.  Compare with #13966.